roles/openshift_health_checker/test/package_availability_test.py

```diff
@@ -3,6 +3,21 @@
 from openshift_checks.package_availability import PackageAvailability
 
 
+@pytest.mark.parametrize('pkg_mgr,is_containerized,is_active', [
+    ('yum', False, True),
+    ('yum', True, False),
+    ('dnf', False, False),
+    ('dnf', False, False),
+])
+def test_is_active(pkg_mgr, is_containerized, is_active):
+    task_vars = dict(
+        ansible_pkg_mgr=pkg_mgr,
+        group_names=["masters", "nodes"],
```

Comment:
This is irrelevant to determine `is_active`, right? If so, we can have one less line to read ;)